syntax/README.md

```diff
@@ -72,3 +72,16 @@ The available annotations right now are all related to vectors:
   fragment[TLSPlaintext.length]`.  Note, however, that in cases where the length
   immediately preceds the array, these can be reframed as vectors with
   appropriate sizes.
+
+
+QUIC Extensions Syntax
+======================
+syntax also supports some minor extensions to allow implementing QUIC.
+
+* The `varint` annotation describes a QUIC-style varint
+* `head=0` means no header, i.e., the bytes are encoded directly on the wire.
```

Comment:
This seems brittle to me -- someone forgets to add a `head` tag things go weird instead of just failing.  Instead, let's define some tokens to go here that indicate what we want, e.g., `head=none`, `head=varint`.  You can map those to const integer values in `tags.go:parseTag()`, as is done for the [`varint` tag itself](https://github.com/ekr/mint/blob/a20fcf6aef8132bc4042baa6a8c7773b0dfe4bc0/syntax/tags.go#L21)

Reply:
And in case it wasn't clear, once you do this, define consts in `tags.go` for the relevant values:

```
const (
  headValueNoHead = 255
  headValueVarint = 254
)
```

That will make the encode/decode functions more readable.